### PR TITLE
Add new font bengali sinhala tamil

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves N/A

**Overall change:**
Adds new fonts for Bengali, Tamil and Sinhala services only for the test environment.

**Code changes:**

- Added new script for Bengali and Sinhala for new fonts
- Added new fonts for Bengali, Tamil and Sinhala
- Added a programmatic check to only apply new fonts on the test environment

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

To test, open dev tools and go to the network tab and click on the font filter tab and reload the page or check against live. Check to see if the new font's are being pulled in. Should be checked in a variety of browsers.

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by amoore108 — APPROVED** [review]

**Comment by amoore108** [comment]
Looks good to me 👍🏼

File: src/app/lib/config/services/bengali.js [hunk 1]
```diff
@@ -4,15 +4,33 @@ import {
   C_GHOST,
   C_POSTBOX_30,
 } from '@bbc/psammead-styles/colours';
-import { bengali } from '@bbc/gel-foundations/scripts';
+import { bengali, bengaliNotoSerif } from '@bbc/gel-foundations/scripts';
 import { bengali as brandSVG } from '@bbc/psammead-assets/svgs';
 import {
   F_SHONAR_BANGLA_BOLD,
   F_SHONAR_BANGLA_REGULAR,
+  F_NOTO_SERIF_BENGALI_BOLD,
```
Comment (jroebu14) [inline src/app/lib/config/services/bengali.js:12]: this might be a very confusing function name. It looks like a string name

**Review by jroebu14 — APPROVED** [review]
LGTM.

Out of scope for this PR but it's unusual that the fonts have to be an array of functions.

File: src/app/lib/config/services/bengali.js (hunk 1 above)
Comment (HarveyPeachey) [reply src/app/lib/config/services/bengali.js:12]: Yeah I found that strange as well

**Comment by HarveyPeachey** [comment]
> LGTM.
> 
> Out of scope for this PR but it's unusual that the fonts have to be an array of functions.

Yeah it looks like we [map over and call the functions](https://github.com/bbc/simorgh/blob/58147fce2e5a51e7725fa104f36cc91d37459129/src/app/Layouts/defaultPageWrapper.jsx#L27-L29) anyway, so I'm not sure why we don't just call the functions and pass on what gets returned.

**Comment by paruchurisilpa** [comment]
**Testing:**
https://www.test.bbc.com/sinhala?renderer_env=live
https://www.test.bbc.com/tamil?renderer_env=live
https://www.test.bbc.com/bengali?renderer_env=live

* Checked using the above links and fonts seems to be rendered correctly.
* ff seems to be showing the Bangla Sangam MN font but correct one is shown in Chrome and Safari..

Screenshot 2021-09-27 at 08 17 56

* On Opera mini extreme mode the font is overlapping for Sinhala.

Screenshot_20210927-094209

* On Opera mini there are spacing issues on Tamil and Bengali but they exist on live as well.

**UX Review:**
* The article author on Sinhala and Tamil is displaying a line-height smaller than the spec. This error is across breakpoints, line-height on article author is too narrow, not based on the spec.

Screenshot 2021-09-24 at 16 45 42
Screenshot 2021-09-24 at 16 52 02

**Comment by paruchurisilpa** [comment]
**Testing:**

* https://www.test.bbc.com/sinhala?renderer_env=live
* https://www.test.bbc.com/tamil?renderer_env=live
* https://www.test.bbc.com/bengali?renderer_env=live

- Fonts are rendered correctly , only in ff its showing Bangla Sangam MN for Bengali where as its fine in Chrome and Safari

Screenshot 2021-09-27 at 08 17 56

- On Opera mini extreme mode the font is overlapping for Sinhala.

Screenshot_20210927-094209

- On Opera mini there are spacing issues on Tamil and Bengali but they exist on live as well.

**UX Review:**
- There is an error across breakpoints, line-height on article author is too narrow, not based on the spec.

Screenshot 2021-09-24 at 16 45 42
Screenshot 2021-09-24 at 16 52 02

**Comment by pvaliani** [comment]
> **Testing:**
> https://www.test.bbc.com/sinhala?renderer_env=live
> https://www.test.bbc.com/tamil?renderer_env=live
> https://www.test.bbc.com/bengali?renderer_env=live
> 
> * Checked using the above links and fonts seems to be rendered correctly.
> * ff seems to be showing the Bangla Sangam MN font but correct one is shown in Chrome and Safari..
> 
> Screenshot 2021-09-27 at 08 17 56
> 

Just addressing comments in turn, updating Firefox appears to resolve the issue with loading in of Noto Serif Bengali - we believe this could be a caching issue or an issue to do with the version of Firefox. Harvey was running version 79.0 64-bit on desktop and was immediately able to load in Noto Serif Bengali font. I updated from 91.0 (where Bangla Sangam NM loaded) to 92.0 and immediately Noto Serif Bengali loaded into the browser:

image

Hope this addresses that issue. @JonBeeb

**Comment by paruchurisilpa** [comment]
I am latest version fo ff 92.0.1 and on an incognito its still showing me Bangla Sangam MN but this is an issue to do with ff rather than from our side, so we can't do anything about it..

Screenshot 2021-09-28 at 09 40 37

Screenshot 2021-09-28 at 10 31 36

**Comment by paruchurisilpa** [comment]
This is now Live. Closing..

**Pull request merged**